spec/spec_helper.rb

```diff
@@ -8,12 +8,11 @@
 
 require 'rspec/rails'
 require 'capybara/rspec'
+require 'database_cleaner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.